bio_curve_fit/logistic.py

```diff
@@ -321,13 +321,32 @@ def predict_inverse(self, y):
         But for samples of unknown concentration, we want to get the concentration as given
         response, which is what this function does.
 
+        args:
```

Comment:
remove (to be consistent with other comments in function; may also want to generally delete trailing spaces in comments or conform them)

Reply:
add ruff numpydoc to enforce consistency